### PR TITLE
Enable the new sensor board for Saluki-FMU2

### PR DESCRIPTION
**Pull request by jlaitine**
Enable the new sensor board for Saluki-FMU2

### CODE REVIEW

**Comment by jlaitine** [comment]
The new sensor board components have been tested on FMU2 + new sensor board. The fixes for the IMU driver are in upstream review, will wait for response from there before merging

File: boards/ssrc/saluki-pi
Comment (jpaali) [inline boards/ssrc/saluki-pi:1]: Replacing icm42688p & icm20649 with icm40609d may cause problems. We have (a lot of) devices with old sensor board. How to handle those? We need detect these somehow.

**Comment by jlaitine** [comment]
Rotated the BMM350 to the correct orientation in saluki-pi submodule. It has moved to underneath the sensor board.

File: boards/ssrc/saluki-pi
Comment (jlaitine) [reply boards/ssrc/saluki-pi:1]: Jaakko Isohella said that the new sensor board is the only one supported by FMU2. So we just drop the support for the old "rigid flex" board.

**Pull request merged**